Overview.bs

```diff
@@ -53,14 +53,13 @@ spec:fetch; type:dfn; for:/; text:response
     "date": "27 Jul 2021"
   },
 
-
-  "OpenType-Variations": {
-    "href": "https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview",
+  "open-type": {
```

Comment:
Specref already has OpenType:

[[OPENTYPE](https://www.specref.org/?q=opentype#)]

Reply:
Specifying our own version of the reference allows me to link to the specific sub pages. The spec ref one links to to default.htm so I can't do relative paths eg. [[open-type/otff#table-directory]]